draft-ietf-rats-architecture.md

```diff
@@ -127,6 +127,100 @@ Appraisal Policy to make application-specific decisions such as authorization de
 The Attestation Result Appraisal Policy might, for example, be configured in the Relying Party 
 by an administrator.
 
+## Composite Attester {#compositeattester}
+
+A Composite Attester is an entity composed of multiple sub-entities such that its
+trustworthiness has to be determined by evaluating all these sub-entities.
+Each sub-entity has at least one Attesting Environment collecting the claims 
+from at least one Target Environment, then this sub-entity generates Evidence
+about its trustworthiness. Therefore each sub-entity can be called an Attester.
+Among these Attesters, there may be only some, which can be called Lead Attesters,
+that have the communication functionality with the Verifier. Other Attesters
+don't have this ability, but they are connected to the Lead Attesters via internal
+links or network connections, and they are evaluated via the Lead Attester's help.
+
+For example, a carrier-grade router is a composite device consisting of a chassis and multiple slots.
+The trustworthiness of the router depends on all its slots' trustworthiness.
+Each slot has an Attesting Environment such as a TPM or TEE collecting the
+claims of its boot process, then it generates Evidence from the claims to prove its trustworthiness.
+Among these slots, only a main slot can communicate with the Verifier
+while other slots cannot. But other slots can communicate with the main
+slot by the links between them inside the router. So the main slot collects
+the Evidence of other slots, produces the final Evidence of the whole router and
+conveys the final Evidence to the Verifier. Therefore the router is a Composite
+Attester, each slot is an Attester and the main slot is the Lead Attester.
+
+Another example is a multi-chassis router which is composed of multiple single carrier-grade routers.
+The multi-chassis router provides higher throughput by interconnecting
+multiple routers and simpler management by being logically treated as one router.
+Among these routers, there is only one main router that connects to the Verifier.
+Other routers are only connected to the main router by the network cables,
+and therefore they are managed and verified via this main router.
+So, in this case, the multi-chassis router is the Composite Attester,
+each router is an Attester and the main router is the Lead Attester.
+
+{{composite}} depicts the data that flows between the Composite Attester and Verifier for the remote attestation.
+
+~~~~
+                   .-----------------------------.
+                   |           Verifier          |
+                   '-----------------------------'
+                                 |  ^
+           Endorsements and      |  | Evidence of
+           Appraisal Policy for  |  | Composite
+           Evidence of Attesters |  | Attester
+                                 |  |
```

Comment:
remove downward line, deal in text with where the policy for the claims collector comes from